resources/ec2-dhcp-options.go

```diff
@@ -7,10 +7,9 @@ import (
 )
 
 type EC2DHCPOption struct {
-	svc        *ec2.EC2
-	id         *string
-	tags       []*ec2.Tag
-	defaultVPC bool
+	svc  *ec2.EC2
+	id   *string
+	tags []*ec2.Tag
```

Comment:
Please keep the `defaultVPC` property. In general I'm unsure why you modified that file, I can't see any improvements.

Reply:
Sorry this was an misstake, it should not be removed. Thanks for the catch.

For some resources it is possible to share them across accounts, for example, with VPC. The VPC then belongs to another account and can actually hopefully only be deleted from this account.

With the OwnerID it is now possible to filter these shared resources. This has the advantage, that there are fewer errors.